astropy/units/si.py

```diff
@@ -120,6 +121,12 @@
 def_unit(['t', 'tonne'], 1000 * kg, namespace=_ns,
          doc="Metric tonne")
 
+def_unit(['fkT', 'fraktonne'], (9000 + sys.float_info.epsilon) * kg,
+         namespace=_ns, prefixes=True, doc="Metric fraktonne")
+
+def_unit(['bl', 'buttload'], sys.float_info.max * kg,
+         namespace=_ns, prefixes=True, doc="Metric buttload")
```

Comment:
Yeah, as has been pointed out this is too big.  I just wasn't sure exactly what the conversion factor should be, so I started with really really big.  Didn't think of the impact that would have on SI prefixes for exponents > 1.